src/components/KonnectorAccount.jsx

```diff
@@ -0,0 +1,54 @@
+import styles from '../styles/accountConnection'
+
+import React from 'react'
+import { translate } from '../plugins/i18n'
+
+import AccountLoginForm from './AccountLoginForm'
+import DescriptionContent from './DescriptionContent'
+
+import { ACCOUNT_ERRORS } from '../lib/accounts'
+
+export const KonnectorAccount = ({ t, connector, account, fields, success, editing, disableSuccessTimeout, oAuthTerminated, submitting, deleting, error, onDelete, onSubmit, onCancel }) => {
+  const { hasDescriptions } = connector
+  const securityIcon = require('../assets/icons/color/icon-cloud-lock.svg')
+
+  const displayHeader = !error && account && editing
```

Comment:
For the first connection we never have an account so the header won't be displayed but we do want to have the descriptions at the first connection as well.